### DISCUSSION TITLE
[Feature Request] Message Box

### DISCUSSION THREAD

**Discussion by zrll12** [heroui-inc/heroui#1813] [Feature requests]
### Is your feature request related to a problem? Please describe.

Currently, we have a Modal component, but this cannot be used for things that is being tracked and may not end soon or it does not need to be confirmed.

### Describe the solution you'd like

1. Call a function
2. Open a modal on the top/bottom/right top/right bottom/left top/left bottom but does not stops user from viewing this page
3. The message box closes by it self after showing a several second or call a close function.

### Describe alternatives you've considered

Maybe a Modal without overlay will work.

### Screenshots or Videos

<img width="418" alt="image" src="https://github.com/nextui-org/nextui/assets/46812903/4d1ce142-d7a9-4ba6-824b-6149000a251c">

**Comment by olehmelnyk** [comment]
This can be implemented with the Card component passed to [sonner](https://sonner.emilkowal.ski/) or [react-hot-toast](https://react-hot-toast.com/docs) library to be rendered as snackbar/notification.

**Reply by zrll12** [reply]
But card in sonner always displays in the bright mode. Changing the theme will not affect components in the toast

**Comment by wingkwong** [comment]
see toast: #314
